so3/kernel/avz/domain_utils.c

```diff
@@ -42,14 +42,14 @@
  * @param agency_fdt_paddr
  * @return
  */
-u64 __get_avz_fdt_paddr(void *agency_fdt_paddr) {
+addr_t __get_avz_fdt_paddr(void *agency_fdt_paddr) {
 	int nodeoffset, next_node;
 	int depth, ret;
-	u64 avz_dt_paddr;
+	addr_t avz_dt_paddr;
 	const char *propstring;
 	bool found = false;
 	volatile char *ptr;
-	addr_t val;
+	u64 val;
```

Comment:
Why is val 64bits even on 32bits systems ?

Reply:
Because U-boot defines all properties of the ITS on 64-bit fields, even if in the .its, the property appears on 32-bit.
u64 on 32-bit arch leads to a long long.